src/content/1.7/development/internationalization/activate-a-language-in-prestashop.md

```diff
@@ -1,32 +1,33 @@
 ---
-title: Make a new language available to be used in the Back Office
+title: Activate a language in PrestaShop
 aliases:
-  - /1.7/development/make_a_new_language_available_to_be_used_in_the_back_office
-  - /1.7/development/make-a-new-language-available-to-be-used-in-the-back-office
+  - /1.7/development/activate_a_language_in_prestashop
+  - /1.7/development/activate-a-language-in-prestashop
 ---
 
-# Make a new language available to be used in the Back Office
+# Activate a language in PrestaShop
 
-Before adding a language in Prestashop, you must check if the language is available in [Crowdin](https://crowdin.com/project/prestashop-official).
+Most languages are available in [Crowdin](https://crowdin.com/project/prestashop-official) for translation, but not all are available in PrestaShop (International > Translations > Add a language).
+Indeed, because of a low completion level on [Crowdin](https://crowdin.com/project/prestashop-official), we do not send the translations to PrestaShop. However, it can smart to add a language if its translators are very active and want to see the results of their Crowdin translations quickly in their own shop.
```

Comment:
```suggestion
Indeed, because of a low completion level on [Crowdin](https://crowdin.com/project/prestashop-official), we do not send the translations to PrestaShop. However, it can be smart to add a language if its translators are very active and want to see the results of their Crowdin translations quickly in their own shop.
```

Reply:
Fixed